website/docs/en/api/modules/module-methods.mdx

```diff
@@ -162,6 +163,14 @@ Tells the browser that the resource might be needed during the current navigatio
 
 A name for the new chunk.
 
+##### webpackFetchPriority
+
+<ApiMeta addedVersion="1.0.0" />
+
+- **Type:**: `"low" | "high" | "auto"`
+
+Set [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority) for specific dynamic imports. It's also possible to set a global default value for all dynamic imports by using the `module.parser.javascript.dynamicImportFetchPriority` option.
```

Comment:
Link to the related config is better

```suggestion
Set [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority) for specific dynamic imports. It's also possible to set a global default value for all dynamic imports by using the [module.parser.javascript.dynamicImportFetchPriority](/config/module#moduleparserjavascriptdynamicimportfetchpriority) option.
```

Reply:
do it with module configuration document